charts/cluster-api-runtime-extensions-nutanix/templates/ccm/nutanix/manifests/helm-addon-installation.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if .Values.hooks.clusterAutoscaler.helmAddonStrategy.defaultValueTemplateConfigMap.create }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: '{{ .Values.hooks.ccm.nutanix.helmAddonStrategy.defaultValueTemplateConfigMap.name }}'
+data:
+  values.yaml: |-
+    ---
+    prismCentralEndPoint: {{ `{{ .PrismCentralHost }}` }}
+    prismCentralPort: {{ `{{ .PrismCentralPort }}` }}
+    prismCentralInsecure: {{ `{{ .PrismCentralInsecure }}` }}
+    prismCentralAdditionalTrustBundle: {{ `"{{ or .PrismCentralAdditionalTrustBundle "" }}"`  }}
```

Comment:
Do we have to specify Prism endpoint related values here?
Can these values template be in the Handler code?
Handler can merge data from `values.yaml` and the parsed template for prism endpoint while creating helm proxy spec.
i.e. keep both the template and parsing logic in the handler code.

Reply:
Thank you Dimitri.